README.md

```diff
@@ -253,16 +251,6 @@ I would like to thank...
 
 ## ❓ FAQ (OPTIONAL) <a name="faq"></a>
```

Comment:
Great job @shuhad786 🤝

- [ ] [Optional] Could you fix the broken MIT license link to make your readme file more professional?
- [ ] [Optional] You may also remove the live demo link since your project is not yet deployed.